src/IssueCreator/Models/IssueMilestone.cs

```diff
@@ -1,23 +1,41 @@
-﻿using System;
+﻿using Octokit;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
 
 namespace IssueCreator.Models
 {
     public class IssueMilestone
     {
-        public int? Number { get; private set; }
-        public string Title { get; private set; }
-        public DateTimeOffset? DueOn { get; private set; }
+        public int? Number { get; set; }
+        public string Title { get; set; }
+        public DateTimeOffset? DueOn { get; set; }
 
-        public IssueMilestone(Octokit.Milestone m)
+        public IssueMilestone(Milestone m)
         {
             Number = m.Number;
             Title = m.Title;
             DueOn = m.DueOn;
         }
 
+        public IssueMilestone() // for deserialization
+        {
+
+        }
+
         public override string ToString()
         {
             return $"{Title}";
         }
+
+        internal static List<IssueMilestone> FromMilestoneList(IReadOnlyList<Milestone> milestones)
```

Comment:
Nit: parameters should use least-derived type required, which in this (and similar) case is `IEnumerable<T>`. Makes it more flexible to refactoring later, and easier to consume across callers.